packages/core/test/test_utils.js

```diff
@@ -37,13 +37,15 @@ export function setProps(comp, newProps) {
   render(React.createElement(comp.constructor, newProps), node.parentNode);
 }
 
-/* Run a group of tests with different combinations of omitExtraData and liveOmit as form props.
+/* Run a group of tests with different combinations of omitExtraData and liveOmit,
+ *  and omitDefaultLoad, as form props.
  */
 export function describeRepeated(title, fn) {
   const formExtraPropsList = [
     { omitExtraData: false },
     { omitExtraData: true },
     { omitExtraData: true, liveOmit: true },
+    { omitDefaultLoad: false },
```

Comment:
Did you mean to do `omitDefaultLoad: true` here? (since it's set to false by default)

Reply:
No; some of the pre-existing tests using this rely on the default behavior. I included it more to be explicit about testing the default behavior, but I can yank it out as it is a bit superfluous.